selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/GolangFileEditingTest.java

```diff
@@ -117,6 +146,13 @@ public void checkCodeValidationFeature() {
     editor.goToCursorPositionVisible(13, 1);
     editor.typeTextIntoEditor(Keys.DELETE.toString());
     editor.waitAllMarkersInvisibility(ERROR);
+
+    // check adding comment code by keyboard 'Ctrl+/'
+    editor.goToCursorPositionVisible(13, 1);
+    editor.launchCommentCodeFeature();
+    editor.waitTextIntoEditor("//package main");
+    editor.launchCommentCodeFeature();
+    editor.waitTextIntoEditor("package main");
```

Comment:
It doesn't actually check that "//" disappear, do it?

Reply:
Yes.. you are right )